web_server/users/static/users/js/mandala.js

```diff
@@ -0,0 +1,263 @@
+const palette = [
```

Comment:
I assume this file and the mandala.css file are all part of that code pen? I would suggest saying so in a comment as well on both these files. We might get hit with plagiarism if we don't